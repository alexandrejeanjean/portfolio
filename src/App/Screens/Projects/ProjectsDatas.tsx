import {
  homeassist1,
  homeassist2,
  homeassist3,
  dipongo1,
  dipongo2,
  dipongo3,
  wog1,
  wog2,
  wog3,
  fillgood1,
  fillgood2,
  fillgood3,
  familyTree,
  familyTree2,
  wineabroad1,
  wineabroad2,
  wineabroad3,
} from '../../Assets/imgPath'

interface Project {
  id: number
  type: string
  title: string
  picture: string
  appType: string
  integration: string
  development: string
  description: string[]
  endpoint: string
  pictures: string[]
  projectUrl: string
}

const ProjectsDatas: Project[] = [
  {
    id: 0,
    type: 'Personnel',
    title: 'WineAbroad',
    picture: 'wineabroad1',
    appType: 'Application web responsive',
    integration: 'Integration',
    development: 'Development',
    description: [
      'WineAbroad est une Single Page Application (SPA) présentant la société et ses services.',
      "WineAbroad est une société spécialisée dans l'export de vins à l'international.",
      "Développée avec Gatsby, pour aider au lancement de l'activité d'un ami, j'ai créé, intégré et développé le projet jusqu'à sa mise en production.",
      '#Gatsby #React #StyledComponent',
    ],
    endpoint: '/projets/wineabroad',
    pictures: [wineabroad1, wineabroad2, wineabroad3],
    projectUrl: 'https://www.wineabroad.fr/',
  },
  {
    id: 1,
    type: 'Professionel',
    title: 'Dipongo',
    picture: 'dipongo',
    appType: 'Application Ios & Android',
    integration: 'Integration',
    development: 'Development',
    description: [
      'Dipongo(v2) est une application crossplateformes mêlant le virtuel au réel.',
      "Stimulant l'imaginaire de l'enfant au travers d'aventures fantastiques.",
      "Développée avec React Native, au sein de l'agence sqal.io, j'ai pris part à l'intégration du responsive pour mobile, et l'ajout de nouvelles fonctionnalités.",
      '#Android #Ios #ReactNative #Redux #Scss',
    ],
    endpoint: '/projets/dipongo',
    pictures: [dipongo1, dipongo2, dipongo3],
    projectUrl: 'https://www.dipongo.net/',
  },
  {
    id: 2,
    type: 'Professionel',
    title: 'Home Assist',
    picture: 'ha',
    appType: 'Application Android',
    integration: 'Integration',
    development: 'Development',
    description: [
      'Home Assist est un application connectée Android qui permet de simplifier et de faciliter le quotidien des seniors vivant seuls et loin de leurs proches.',
      "En collaboration avec le développeur back-end de chez Home Assist, j'ai réalisé au sein de l'agence sqal.io, l'intégration des maquettes. Dans un second temps le développement de l'application Android, ainsi qu'une application mobile à destination des proches.",
      '#Android #ReactNative #Redux #StyledComponent #Firebase',
    ],
    endpoint: '/projets/home-assist',
    pictures: [homeassist1, homeassist2, homeassist3],
    projectUrl: 'http://www.home-assist.fr/',
  },
  {
    id: 3,
    type: 'Professionel',
    title: 'War of Golf',
    picture: 'wog',
    appType: 'Application Ios & Android',
    integration: 'Integration',
    development: 'Development',
    description: [
      "War of Golf est une application Android et Ios qui offre une experience nouvelle du golf. Affronter ses amis sur le terrain, créer sa communauté, ou apprendre à jouer au golf via l'Academy.",
      "Au sein de l'agence sqal.io, il m'a été confié, la refonte intégrale et le développement de nouvelles fonctionnalités de la version 2.0.",
      '#Android #Ios #ReactNative #Redux #StyledComponent #WebSocket #GoogleAdmob #Firebase #OneSignal',
    ],
    endpoint: '/projets/war-of-golf',
    pictures: [wog1, wog2, wog3],
    projectUrl: 'https://warofgolf.com/',
  },
  {
    id: 4,
    type: 'Professionel',
    title: 'Fillgood',
    picture: 'fillgood',
    appType: 'Application web responsive',
    integration: 'Integration',
    development: 'Development',
    description: [
      "L'application web FillGood, est une plateforme permettant la livraison de carburant aux particuliers et entreprises.",
      "En charge de l'intégration et du développement de l'application web au sein de l'agence sqal.io. J'ai développé une interface entièrement responsive, avec 3 accès : administrateur, employé et client.",
      '#Web #ReactJS #Redux #StyledComponent #Responsive',
    ],
    endpoint: '/projets/fillgood',
    pictures: [fillgood1, fillgood2, fillgood3],
    projectUrl: 'https://fill-good.fr/',
  },
  {
    id: 5,
    type: 'Personnel',
    title: 'Family Tree',
    picture: 'familyTree',
    appType: 'Application web',
    integration: 'Integration',
    development: 'Development',
    description: [
      "L'application web Family Tree, est une application responsive, créée dans un but de montée en compétences fullstack.",
      "L'utilisateur peut créer un compte, ajouter sa famille, les générations et les membres qui la compose.",
      "Pour réaliser cette application, j'ai imaginé le design, l'ai intégré, puis réalisé le développement de l'ensemble des composants, jusqu'aux tests unitaires.",
      'Je me suis chargé de créer une API REST en Node/Express, qui permet de servir le client.',
      'Je poursuis actuellement le développement, pour la partie client et server afin de présenter ce projet comme réalisation dans mon portfolio.',
      '#Web #Responsive #ReactJS #scss #bootstrap #Jest #Enzyme #API #NodeJS #Express #Sequelize',
    ],
    endpoint: '/projets/familytree',
    pictures: [familyTree, familyTree2],
    projectUrl: 'https://family-tree-app-app.herokuapp.com/',
  },
]

export default ProjectsDatas
