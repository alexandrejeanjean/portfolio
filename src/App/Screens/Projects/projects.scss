@import '../../constants/colors.scss';

$overlay-dipongo: '../../Assets/project-overlay-dipongo.svg';
$overlay-wog: '../../Assets/project-overlay-wog.svg';
$overlay-ha: '../../Assets/project-overlay-ha.svg';
$overlay-fg: '../../Assets/project-overlay-fg.svg';

.projects-bubble {
  position: fixed;
  bottom: -15%;
  left: 10%;
  right: 0;
  margin: auto;
  width: 100%;
  z-index: -1;
  animation-name: onLoad;
  animation-duration: 0.8s;
  animation-fill-mode: forwards;
}

.projects-wrapper {
  padding-bottom: 2%;
  .projects-title-wrapper {
    padding-left: 100px;
    display: block;
    h1 {
      font-size: 4rem;
      margin-bottom: 10px;

      .underlined {
        font: inherit;
        color: inherit;
        position: relative;
      }
    }
    p {
      max-width: 40%;
      line-height: 30px;
    }
  }
  .projects-cards-wrapper {
    margin-top: 2%;
    padding-left: 100px;
    padding-right: 200px;
    display: grid;
    grid-template-columns: repeat(auto-fit, minmax(450px, 1fr));
    grid-gap: 50px 50px;
    grid-auto-flow: dense;

    .dipongo-bg {
      background-image: url('../../Assets/projects/dipongo1.png');
    }

    .wog-bg {
      background-image: url('../../Assets/projects/wog2.png');
    }

    .ha-bg {
      background-image: url('../../Assets/projects/homeassist2.png');
    }
    .fg-bg {
      background-image: url('../../Assets/projects/fillgood3.png');
    }

    .project-card {
      width: 450px;
      height: 400px;
      background-position: center;
      background-size: cover;
      border-radius: 10px;
      padding: 8%;
      transition: all 0.3s cubic-bezier(0.25, 0.8, 0.25, 1);
      box-shadow: 0 10px 20px rgba(0, 0, 0, 0.19), 0 6px 6px rgba(0, 0, 0, 0.23);
      position: relative;
      filter: grayscale(0.7);
      animation-name: onLoad;
      animation-duration: 1.4s;
      &:hover {
        filter: grayscale(0);
        transform: translateY(2px);
        box-shadow: 0 19px 38px rgba(0, 0, 0, 0.3),
          0 15px 12px rgba(0, 0, 0, 0.22);
      }

      .card-datas-wrapper {
        position: absolute;
        bottom: 0;
        z-index: 3;
        padding-bottom: 3%;
        .card-title {
          max-width: 100%;
          margin: 0;
          font-family: 'NeutraTextBold', 'Arial';
          color: $main;
          font-size: 2.8rem;
        }
        .card-meta {
          display: block;
          list-style-type: none;
          display: flex;
          justify-content: flex-start;
          padding: 0;
          color: $main;
          margin-top: 6px;
          .card-meta-item:first-child {
            position: relative;
            margin-right: 12px;
            &::after {
              content: '|';
              position: absolute;
              top: 0;
              right: -8px;
            }
          }
        }
        .meta-right {
          justify-content: flex-end;
        }
      }
      .title-right {
        right: 5%;
      }
    }

    .left.overlay-dipongo::after {
      background-image: url(#{$overlay-dipongo});
      background-repeat: no-repeat;
    }
    .right.overlay-fg::after {
      background-image: url(#{$overlay-fg});
      background-repeat: no-repeat;
    }
    .right.overlay-ha::after {
      background-image: url(#{$overlay-ha});
      background-repeat: no-repeat;
    }
    .left.overlay-wog::after {
      background-image: url(#{$overlay-wog});
      background-repeat: no-repeat;
    }

    .left {
      margin-top: 10%;
      position: relative;
      overflow-y: hidden;
      padding-bottom: 5%;
      font-family: 'NeutraTextBook', 'Arial';
      color: $main;
      &:hover::before {
        max-width: 70%;
        content: attr(data-description);
        white-space: pre-wrap;
        line-height: 30px;
        position: absolute;
        top: 25%;
        left: 10%;
        z-index: 3;
        opacity: 0;
        animation-name: textApparition;
        animation-delay: 0.8s;
        animation-fill-mode: forwards;
        animation-duration: 1s;
      }
      &::after {
        content: '';
        width: 100%;
        height: 100%;
        position: absolute;
        left: -9rem;
        top: 12rem;
        z-index: 2;
        display: inline-block;
      }

      &:hover::after {
        animation: sheenLeft 1s forwards;
      }
    }

    .right {
      position: relative;
      overflow-y: hidden;
      padding-bottom: 5%;
      font-family: 'NeutraTextBook', 'Arial';
      color: $main;
      &:hover::before {
        max-width: 70%;
        content: attr(data-description);
        line-height: 30px;
        position: absolute;
        top: 30%;
        left: 10%;
        z-index: 2;
        opacity: 0;
        animation-name: textApparition;
        animation-delay: 0.8s;
        animation-fill-mode: forwards;
        animation-duration: 1s;
      }
      &::after {
        content: '';
        width: 100%;
        height: 100%;
        position: absolute;
        top: 12rem;
        right: -10rem;
        z-index: 1;
        display: inline-block;
        &:hover {
          width: 300%;
          height: 300%;
        }
      }

      &:hover::after {
        animation: sheenRight 1s forwards;
      }
    }
  }

  .contact-btn-wrapper {
    width: 100%;
    margin-top: 5%;
    display: flex;
    justify-content: center;
  }
}

@media screen and (max-width: 768px) {
  .projects-wrapper {
    padding-bottom: 2%;

    .projects-title-wrapper {
      padding-left: 0;
      display: block;
      h1 {
        font-size: 3rem;
        margin-bottom: 10px;
      }
      p {
        max-width: 100%;
        line-height: 30px;
      }
    }

    .projects-cards-wrapper {
      margin-top: 2%;
      padding-left: 0;
      padding-right: 0;
      grid-gap: 50px 50px;

      .project-card {
        width: 85vw;
        height: 400px;
        background-position: center;
        background-size: cover;
        border-radius: 10px;
        padding: 0%;

        &:hover {
          box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16),
            0 3px 6px rgba(0, 0, 0, 0.23);
        }

        .card-datas-wrapper {
          padding: 2%;
          .card-title {
            font-size: 2rem;
          }
        }
      }
      .left {
        margin-top: 0;
        &:hover::after,
        &:focus::after {
          display: block;
          content: '';
          width: 100%;
          height: 100%;
          position: absolute;
          top: 0;
          left: 0;
          z-index: 1;

          animation-name: sheenMobile;
          animation-duration: 0.6s;
          background-color: rgba(255, 255, 255, 0);
          background-image: none;
        }
        &:hover::before,
        &:focus::before {
          line-height: 25px;
          max-width: 85%;
          top: 5%;
          left: 7.5%;
        }
      }

      .right {
        margin-top: 0;
        &:hover::after,
        &:focus::after {
          display: block;
          content: '';
          width: 100%;
          height: 100%;
          position: absolute;
          top: 0;
          right: 0;
          z-index: 1;
          animation-name: sheenMobile;
          animation-duration: 0.6s;
          background-color: rgba(255, 255, 255, 0);
          background-image: none;
        }
        &:hover::before,
        &:focus::before {
          line-height: 25px;
          max-width: 85%;
          top: 5%;
          left: 7.5%;
        }
      }
    }

    .right.overlay-fg::after,
    .right.overlay-ha::after,
    .left.overlay-dipongo::after,
    .left.overlay-wog::after {
      display: none;
    }
  }

  .contact-btn-wrapper {
    padding-bottom: 120px;
  }
}

@keyframes textApparition {
  0% {
    opacity: 0;
  }
  100% {
    opacity: 1;
  }
}

@keyframes sheenRight {
  100% {
    transform: rotateZ(60deg) translate(0, -20%);
    width: 200%;
    height: 200%;
  }
}

@keyframes sheenLeft {
  100% {
    transform: rotateZ(55deg) translate(-14%, 2%);
    width: 200%;
    height: 200%;
  }
}

@keyframes sheenMobile {
  100% {
    background-color: rgba(255, 255, 255, 0.8);
  }
}
