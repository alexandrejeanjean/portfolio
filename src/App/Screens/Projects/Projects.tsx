import React from 'react'
import classNames from 'classnames'
import './projects.scss'
// import BubbleT from '../../Assets/BubbleT.svg'
import ProjectsDatas from './ProjectsDatas'
import ClassicBtn from '../../Shared/Button/Button'

let cardStyle = (
  i: number,
  isTitle: boolean,
  isMetaTitle: boolean,
  picture: string
) => {
  if (isTitle) {
    return classNames({
      'card-datas-wrapper': true,
      'title-right': isTitle && i % 2 === 1,
    })
  }

  if (isMetaTitle) {
    return classNames({
      'card-meta': true,
      'meta-right': i % 2 === 0,
    })
  }

  return classNames({
    'project-card': true,
    left: i % 2 === 0,
    right: i % 2 === 1,
    'dipongo-bg': picture === 'dipongo',
    'wog-bg': picture === 'wog',
    'ha-bg': picture === 'ha',
    'fg-bg': picture === 'fillgood',
    'overlay-dipongo': i % 2 === 0 && picture === 'dipongo',
    'overlay-ha': i % 2 === 1 && picture === 'ha',
    'overlay-wog': i % 2 === 0 && picture === 'wog',
    'overlay-fg': i % 2 === 1 && picture === 'fillgood',
  })
}

const Projects = () => {
  return (
    <>
      {/* <img src={BubbleT} className='projects-bubble' alt='' /> */}
      <main className='projects-wrapper page'>
        <section className='projects-title-wrapper'>
          <h1>
            Sélection de <span className='underlined'>projets.</span>
          </h1>
          <p>
            Je vous propose ici un échantillon des projets auxquels j'ai pris
            part depuis mes premiers pas dans le web jusqu'à aujourd'hui.
          </p>
        </section>
        <section className='projects-cards-wrapper'>
          {ProjectsDatas.map((item, i) => (
            <div
              className={cardStyle(i, false, false, item.picture)}
              data-description={item.description[0]}
              data-description2={item.description[1]}
              data-description3={item.description[2]}
              key={item.title}
            >
              <div className={cardStyle(i, true, false, item.picture)}>
                <h2 className='card-title'>{item.title}</h2>
                <ul className={cardStyle(i, false, true, item.picture)}>
                  <li className='card-meta-item'>{item.integration}</li>
                  <li>{item.development}</li>
                </ul>
              </div>
            </div>
          ))}
        </section>
        <div className='contact-btn-wrapper'>
          <ClassicBtn text='Ecrivez-moi' endpoint='/' />
        </div>
      </main>
    </>
  )
}

export default Projects
