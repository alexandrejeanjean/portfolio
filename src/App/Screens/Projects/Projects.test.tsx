import React from 'react'
import { shallow } from 'enzyme'

import Projects from './Projects'

let ProjectsWrapper: any

beforeEach(() => {
  ProjectsWrapper = shallow(<Projects />)
})

describe('Projects component', () => {
  it('should render Projects component correctly', () => {
    expect(ProjectsWrapper).toMatchSnapshot()
  })
})
