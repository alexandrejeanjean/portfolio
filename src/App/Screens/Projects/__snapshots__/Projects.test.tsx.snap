// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects component should render Projects component correctly 1`] = `
<Fragment>
  <main
    className="projects-wrapper page"
  >
    <section
      className="projects-title-wrapper"
    >
      <h1>
        Sélection de 
        <span
          className="underlined"
        >
          projets.
        </span>
      </h1>
      <p>
        Je vous propose ici un échantillon des projets auxquels j'ai pris part depuis mes premiers pas dans le web jusqu'à aujourd'hui.
      </p>
    </section>
    <section
      className="projects-cards-wrapper"
    >
      <Link
        className="project-card-wrapper"
        key="Dipongo"
        to={
          Object {
            "pathname": "/projets/infos",
            "state": Object {
              "project": Object {
                "appType": "Application Ios & Android",
                "description": Array [
                  "Dipongo(v2) est une application crossplateformes mêlant le virtuel au réel.",
                  "Stimulant l'imaginaire de l'enfant au travers d'aventures fantastiques.",
                  "Développée avec React Native, au sein de l'agence sqal.io, j'ai pris part à l'intégration du responsive pour mobile, et l'ajout de nouvelles fonctionnalités.",
                  "#Android #Ios #ReactNative #Redux #Scss",
                ],
                "development": "Development",
                "endpoint": "/projets/dipongo",
                "id": 0,
                "integration": "Integration",
                "picture": "dipongo",
                "pictures": Array [
                  null,
                  null,
                  null,
                ],
                "projectUrl": "https://www.dipongo.net/",
                "title": "Dipongo",
                "type": "Professionel",
              },
            },
          }
        }
      >
        <div
          className="project-card left dipongo-bg"
        />
        <div
          className="card-datas-wrapper title-left"
        >
          <h2
            className="card-title"
          >
            Dipongo
          </h2>
          <ul
            className="card-meta meta-left"
          >
            <li
              className="card-meta-item"
            >
              Integration
            </li>
            <li>
              Development
            </li>
          </ul>
        </div>
        <div
          className="project-overlay"
        />
        <div
          className="project-card-button left"
        />
      </Link>
      <Link
        className="project-card-wrapper"
        key="Home Assist"
        to={
          Object {
            "pathname": "/projets/infos",
            "state": Object {
              "project": Object {
                "appType": "Application Android",
                "description": Array [
                  "Home Assist est un application connectée Android qui permet de simplifier et de faciliter le quotidien des seniors vivant seuls et loin de leurs proches.",
                  "En collaboration avec le développeur back-end de chez Home Assist, j'ai réalisé au sein de l'agence sqal.io, l'intégration des maquettes. Dans un second temps le développement de l'application Android, ainsi qu'une application mobile à destination des proches.",
                  "#Android #ReactNative #Redux #StyledComponent #Firebase",
                ],
                "development": "Development",
                "endpoint": "/projets/home-assist",
                "id": 1,
                "integration": "Integration",
                "picture": "ha",
                "pictures": Array [
                  null,
                  null,
                  null,
                ],
                "projectUrl": "http://www.home-assist.fr/",
                "title": "Home Assist",
                "type": "Professionel",
              },
            },
          }
        }
      >
        <div
          className="project-card right ha-bg"
        />
        <div
          className="card-datas-wrapper title-right"
        >
          <h2
            className="card-title"
          >
            Home Assist
          </h2>
          <ul
            className="card-meta meta-right"
          >
            <li
              className="card-meta-item"
            >
              Integration
            </li>
            <li>
              Development
            </li>
          </ul>
        </div>
        <div
          className="project-overlay"
        />
        <div
          className="project-card-button"
        />
      </Link>
      <Link
        className="project-card-wrapper"
        key="War of Golf"
        to={
          Object {
            "pathname": "/projets/infos",
            "state": Object {
              "project": Object {
                "appType": "Application Ios & Android",
                "description": Array [
                  "War of Golf est une application Android et Ios qui offre une experience nouvelle du golf. Affronter ses amis sur le terrain, créer sa communauté, ou apprendre à jouer au golf via l'Academy.",
                  "Au sein de l'agence sqal.io, il m'a été confié, la refonte intégrale et le développement de nouvelles fonctionnalités de la version 2.0.",
                  "#Android #Ios #ReactNative #Redux #StyledComponent #WebSocket #GoogleAdmob #Firebase #OneSignal",
                ],
                "development": "Development",
                "endpoint": "/projets/war-of-golf",
                "id": 2,
                "integration": "Integration",
                "picture": "wog",
                "pictures": Array [
                  null,
                  null,
                  null,
                ],
                "projectUrl": "https://warofgolf.com/",
                "title": "War of Golf",
                "type": "Professionel",
              },
            },
          }
        }
      >
        <div
          className="project-card left wog-bg"
        />
        <div
          className="card-datas-wrapper title-left"
        >
          <h2
            className="card-title"
          >
            War of Golf
          </h2>
          <ul
            className="card-meta meta-left"
          >
            <li
              className="card-meta-item"
            >
              Integration
            </li>
            <li>
              Development
            </li>
          </ul>
        </div>
        <div
          className="project-overlay"
        />
        <div
          className="project-card-button left"
        />
      </Link>
      <Link
        className="project-card-wrapper"
        key="Fillgood"
        to={
          Object {
            "pathname": "/projets/infos",
            "state": Object {
              "project": Object {
                "appType": "Application web",
                "description": Array [
                  "L'application web FillGood, est une plateforme permettant la livraison de carburant aux particuliers et entreprises.",
                  "En charge de l'intégration et du développement de l'application web au sein de l'agence sqal.io. J'ai développé une interface entièrement responsive, avec 3 accès : administrateur, employé et client.",
                  "#Web #ReactJS #Redux #StyledComponent #Responsive",
                ],
                "development": "Development",
                "endpoint": "/projets/fillgood",
                "id": 3,
                "integration": "Integration",
                "picture": "fillgood",
                "pictures": Array [
                  null,
                  null,
                  null,
                ],
                "projectUrl": "https://fill-good.fr/",
                "title": "Fillgood",
                "type": "Professionel",
              },
            },
          }
        }
      >
        <div
          className="project-card right fg-bg"
        />
        <div
          className="card-datas-wrapper title-right"
        >
          <h2
            className="card-title"
          >
            Fillgood
          </h2>
          <ul
            className="card-meta meta-right"
          >
            <li
              className="card-meta-item"
            >
              Integration
            </li>
            <li>
              Development
            </li>
          </ul>
        </div>
        <div
          className="project-overlay"
        />
        <div
          className="project-card-button"
        />
      </Link>
      <Link
        className="project-card-wrapper"
        key="Family Tree"
        to={
          Object {
            "pathname": "/projets/infos",
            "state": Object {
              "project": Object {
                "appType": "Application web",
                "description": Array [
                  "L'application web Family Tree, est une application responsive, créée dans un but de montée en compétences fullstack.",
                  "L'utilisateur peut créer un compte, ajouter sa famille, les générations et les membres qui la compose.",
                  "Pour réaliser cette application, j'ai imaginé le design, l'ai intégré, puis réalisé le développement de l'ensemble des composants, jusqu'aux tests unitaires.",
                  "Je me suis chargé de créer une API REST en Node/Express, qui permet de servir le client.",
                  "Je poursuis actuellement le développement, pour la partie client et server afin de présenter ce projet comme réalisation dans mon portfolio.",
                  "#Web #Responsive #ReactJS #scss #bootstrap #Jest #Enzyme #API #NodeJS #Express #Sequelize",
                ],
                "development": "Development",
                "endpoint": "/projets/familytree",
                "id": 4,
                "integration": "Integration",
                "picture": "familyTree",
                "pictures": Array [
                  null,
                  null,
                ],
                "projectUrl": "https://family-tree-app-app.herokuapp.com/",
                "title": "Family Tree",
                "type": "Personnel",
              },
            },
          }
        }
      >
        <div
          className="project-card left familytree-bg"
        />
        <div
          className="card-datas-wrapper title-left"
        >
          <h2
            className="card-title"
          >
            Family Tree
          </h2>
          <ul
            className="card-meta meta-left"
          >
            <li
              className="card-meta-item"
            >
              Integration
            </li>
            <li>
              Development
            </li>
          </ul>
        </div>
        <div
          className="project-overlay"
        />
        <div
          className="project-card-button left"
        />
      </Link>
    </section>
    <div
      className="contact-btn-wrapper"
    >
      <ClassicButton
        endpoint="/"
        isMailTo={true}
        text="Ecrivez-moi"
      />
    </div>
  </main>
</Fragment>
`;
